src/types.ts

```diff
@@ -190,6 +190,8 @@ export namespace CredentialedActionShape {
   export type AutomaticGameEvent = ReturnType<
     typeof ActionCreators.automaticGameEvent
   >;
+  export type Undo = ReturnType<typeof ActionCreators.undo>;
+  export type Redo = ReturnType<typeof ActionCreators.redo>;
   export type Any =
```

Comment:
Can you also use the new `Undo / Redo` types in `Any` (rather than the credential-less versions being used there currently)?